ember/app/styles/components/_nav-bar.scss

```diff
@@ -28,3 +28,24 @@
     }
   }
 }
+
+.language-select {
+  .uk-dropdown {
+    min-width: 100px;
+  }
+}
+
+.language-select-offcanvas {
+  .uk-dropdown {
+    min-width: 100px;
+    max-width: 100px;
+  }
+
+  ul {
+    background: #222;
+
+    li a {
+      color: rgba(255, 255, 255, 0.7);
```

Comment:
What is the reason for this specific selector? Do you need a non-uikit color?

Reply:
The problem lies with the off canvas colors, the background would be white but the text as well. This selector is so that the text color is consistent.

Reply:
Isn't there a uikit variable for this text color?

Reply:
Yes there is one